Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -152,11 +152,11 @@ class ConfigurationManager
     protected $cacheNeedsUpdate = false;
 
     /**
-     * An absolute file path to store configuration caches in. If null no cache will be active.
+     * An absolute file path to store configuration caches in. If not set, no cache will be active.
      *
-     * @var string
+     * @var ?string
```

Comment:
this is no valid php doctype - in doctypes there are only union types - so there it should be `string|null`

Reply:
well i didn't know that actually thanks ^^